TestSuite/integration.py

```diff
@@ -56,16 +56,15 @@ def build(
         if image is not None:
             self.image.write_bytes(image)
 
-    def create_default_integration(self, name: str = '', commands: str = ''):
+    def create_default_integration(self, name: str = None, commands: List[str] = None):
```

Comment:
Add docstrings to all "create default" functions, now that there is real logic happening.

Reply:
That's the convention for UT docstrings, not for used functions.
Please change it.

Reply:
@MosheEichler Please use google docstring: 
![image](https://user-images.githubusercontent.com/50294648/122909386-d0986680-d35d-11eb-808f-b42d7f9bc2fe.png)

you can read about it here
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
```suggestion
    def create_default_integration(self, name: str = 'Sample', commands: List[str] = None):
```